gcloud/datastore/connection.py

```diff
@@ -317,9 +304,9 @@ def commit(self, project, request_pb):
         request_pb.project_id = project
         try:
             return self._stub.Commit(request_pb, GRPC_TIMEOUT_SECONDS)
```

Comment:
You can pass a `grpc.CallCredentials` on every call, but if it's just going to be the same `grpc.CallCredentials` on every call, I think composing the `grpc.CallCredentials` with the `grpc.ChannelCredentials` at `grpc.Channel`-creation time has the same effect and is slightly more attractive.

What's attracting you to passing your `grpc.CallCredentials` value at each RPC invocation?